### PR TITLE
gha: update: don't create PR if version is null

### PR DESCRIPTION
**Pull request by efiop**

### CODE REVIEW

File: .github/workflows/update.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
         ./update.sh ${{ steps.latest.outputs.version }}
         echo "changes=$(git diff | tr '\n' ' ')" >> $GITHUB_OUTPUT
     - name: create PR
-      if: ${{ steps.update.outputs.changes != '' }}
+      if: ${{ steps.update.outputs.changes != '' && steps.latest.outputs.version != 'null' }}
```
Comment (shcheklein) [inline .github/workflows/update.yaml:26]: I think we should fail (or retry + fail) in this case (not silently ignore and move on). We don't want to wait another day for the release (it's already usually take quite a long time).

Comment (efiop) [reply .github/workflows/update.yaml:26]: @shcheklein It is not worth an exponential backoff 😆

Comment (efiop) [reply .github/workflows/update.yaml:26]: snap is not essential, waiting for a day is not a big deal. This is on-a-surface fix to remove our headache. Can make cron job run more often, but I don't see a point tbh.

Comment (efiop) [reply .github/workflows/update.yaml:26]: We could use gh CLI for this these days, I guess. Was trying to move on, but I guess I'll take a look...

Comment (shcheklein) [reply .github/workflows/update.yaml:26]: I see your point, but if snap is not important I would completely drop it then. If we maintain and support and have some users we should respect that and make basic effort to optimize cycle to ship (esp when p0 happens, or like we have now - a discrepancy between DVC and VS Code, etc, etc.).

Overall, I would say, even current state is better then vs this update. We see an email with `dvc null` at least.

Comment (efiop) [reply .github/workflows/update.yaml:26]: wNot sure I follow, what email?

Comment (shcheklein) [reply .github/workflows/update.yaml:26]: sorry for the confusion, I mean GH notifications (in my case they come also to my inbox in some cases).

Comment (efiop) [reply .github/workflows/update.yaml:26]: https://github.com/iterative/dvc-snap/pull/120

**Pull request closed without merging**